docs/coordinates/index.rst

```diff
@@ -341,7 +340,10 @@ a quick way to get an `~astropy.coordinates.EarthLocation` - the
     <EarthLocation (-1463969.30185172, -5166673.34223433, 3434985.71204565) m>
 
 To see the list of site names available, use
-:func:`astropy.coordinates.EarthLocation.get_site_names`.
+:func:`~astropy.coordinates.EarthLocation.get_site_names`::
+
+    >>> EarthLocation.get_site_names()  # doctest: +REMOTE_DATA
```

Comment:
```suggestion
    >>> EarthLocation.get_site_names()  # doctest: +SKIP
```

Reply:
I uncovered the problem this pull request is meant to solve by implementing this test, so it clearly serves a purpose and I'd rather not skip it.

Reply:
I agree with @eerovaher here!